src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid $cart-border-color;
+  border-radius: 5px;
+  background-color: $cart-bg-color;
+  padding: $card-padding;
+
+  &__product-img {
+    display: flex;
+    justify-content: center;
+    padding: $product-img-padding 0;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    font-weight: 500;
+    line-height: 18px;
+    color: $page-text-color;
+    margin: $product-img-padding + 8 0 4px;
+  }
+
+  &__code {
+    color: $gray-color;
+    font-size: $font-S;
+    margin: 0;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 8px 0 $product-img-padding;
+
+    &-txt {
+      color: $gray-color;
+      margin: 0;
+    }
+
+    &-sum {
+      color: $page-text-color;
+      font-size: 16px;
+      font-weight: 700;
+      margin: 0;
+      line-height: 18px;
+    }
+  }
```

Comment:
The font size is missing for `.card__price-txt`. It's always a good practice to specify the font size for text elements.